### PR TITLE
MOI 1.0

### PR DESCRIPTION
**Pull request by joaquimg**

### CODE REVIEW

**Review by odow — APPROVED** [review]

**Comment by joaquimg** [comment]
@odow I noted you still want to do "things" in gurobi em cplex. Should we consider some "things" for xpress?

**Comment by odow** [comment]
Gurobi and CPLEX still have excluded tests, like the infeasibility certificate stuff. And they're missing a copy-to thing. I also don't know if we should tag a 1.0 without official support. (It's an incentive. Hey, your software is still in beta)

**Comment by joaquimg** [comment]
I would be fine not tagging this as 1.0

File: Project.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name = "Xpress"
 uuid = "9e70acf3-d6c9-5be6-b5bd-4e2c73e3e054"
 authors = ["Joaquim Dias Garcia <joaquim@psr-inc.com>, Dheepak Krishnamurthy <me@kdheepak.com>, Jose Daniel Lara <jdlara@berkeley.edu>"]
 url = "https://github.com/jump-dev/Xpress.jl"
-version = "0.14.0"
+version = "1.0.0"
```
Comment (joaquimg) [inline Project.toml:5]:
```suggestion
version = "0.15.0"
```

**Pull request merged**